.github/workflows/integrationTest.yml

```diff
@@ -564,6 +570,7 @@ jobs:
               -var="binary_name=${{ matrix.arrays.binaryName }}" \
               -var="local_stack_host_name=${{ needs.StartLocalStack.outputs.local_stack_host_name }}" \
               -var="s3_bucket=${S3_INTEGRATION_BUCKET}" \
+              -var="plugin_tests"='${{ github.event.inputs.plugins }}'" \
```

Comment:
Will this be empty on merge or throw an error?

Reply:
it'll be empty cause the default is `''`